tests/unit/test_tf_estimator.py

```diff
@@ -738,6 +738,12 @@ def test_script_mode_deprecated_args(sagemaker_session):
     assert _deprecated_args_msg('training_steps, evaluation_steps, requirements_file, checkpoint_path') in str(e.value)
 
 
+def test_legacy_mode_deprecated(sagemaker_session):
```

Comment:
should we add a test for the opposite case?

Reply:
will do